project_national/fact_sheets_category_2.yml

```diff
@@ -0,0 +1,492 @@
+      - &ducts_category3 #improving leakage some, and insulation
+        option: Ducts|10% Leakage, R-8
+        apply_logic:
+          or: 
+            - Ducts|20% Leakage, Uninsulated
+            - Ducts|20% Leakage, R-4
+            - Ducts|20% Leakage, R-6 
+        costs:
+          - value: 2.32 # Mid-value from REMDB, inflated from 2010 to 2022 dollars using 1.289 factor
+            multiplier: Duct Unconditioned Surface Area (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      - &ducts_category4 #improving leakage some only
+        option: Ducts|10% Leakage, R-8
+        apply_logic:
+          or: 
+            - Ducts|20% Leakage, R-8
+        costs:
+          - value: 0.52 # Mid-value from REMDB, inflated from 2010 to 2022 dollars using 1.289 factor
+            multiplier: Duct Unconditioned Surface Area (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      #ducts categories 5 & 6 from ABC work do not apply to any duct configurations in the current ResStock baseline
+        # - &ducts_category5 #improving leakage a little, and insulation
+        #   option: Ducts|10% Leakage, R-8
+        #   apply_logic:
+        #     or: 
+        #       - Ducts|15% Leakage, Uninsulated
+        #       - Ducts|15% Leakage, R-4
+        #       - Ducts|15% Leakage, R-6
+        #       - Ducts|14% Leakage, Uninsulated
+        #       - Ducts|14% Leakage, R-4
+        #       - Ducts|14% Leakage, R-6
+        #   costs:
+        #     - value: 2.06 # Mid-value from REMDB, inflated from 2010 to 2022 dollars using 1.289 factor
+        #       multiplier: Duct Unconditioned Surface Area (ft^2)
+        #   lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+        # - &ducts_category6 #improving leakage a little only
+        #   option: Ducts|10% Leakage, R-8
+        #   apply_logic:
+        #     or: 
+        #       - Ducts|15% Leakage, R-8
+        #       - Ducts|14% Leakage, R-8
+        #   costs:
+        #     - value: 0.26 # Mid-value from REMDB, inflated from 2010 to 2022 dollars using 1.289 factor
+        #       multiplier: Duct Unconditioned Surface Area (ft^2)
+        #   lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      - &ducts_category7 #improving insulation only
+        option: Ducts|10% Leakage, R-8
```

Comment:
Any option not addressed is `- Ducts|0% Leakage, Uninsulated` (~10%)

Reply:
The upgrade value of leakage is 10%. It may not make sense to upgrade ducts from _0% Leakage, Uninsulated_ to _10% Leakage, R-8_. Because the leakage is increased after upgrade.

Reply:
We could have an option to replace `- Ducts|0% Leakage, Uninsulated`  to `- Ducts|0% Leakage, R-8`

Reply:
@ekpresent

Reply:
Yes, not sure why we didn't include this in EUSS RR1. In all cases insulation should be brought up to R-8 and leakage should be brought down to 10%. If one or the other already exceeds, it should be maintained as-is. So Ducts|0% Leakage, Uninsulated should be improved to Ducts|0% Leakage, R-8 as Lixi suggested.

Reply:
done